### PR TITLE
docker tweaks

### PR DESCRIPTION
**Pull request by r-tx**
* Install MVT from github rather than PiPY
* Install latest abe

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -57,12 +57,12 @@ RUN git clone https://github.com/libimobiledevice/libplist \
 
 # Installing MVT
 # --------------
-RUN pip3 install mvt
+RUN pip3 install git+https://github.com/mvt-project/mvt.git@main
```
Comment (roaree) [inline Dockerfile:60]: Ideally I think we should be copying in from whatever's in the locally checked out directory, but this seems like an improvement regardless

**Review by roaree — APPROVED** [review]
Thanks!

File: Dockerfile (hunk 1 above)
Comment (r-tx) [reply Dockerfile:60]: I could update it for that,  though I wonder would it make more sense for you to put it up on docker hub?  It'd cut down on the steps a little to get the container up and running.

Comment (roaree) [reply Dockerfile:60]: I would be in favour of us publishing an official docker image

**Pull request merged**